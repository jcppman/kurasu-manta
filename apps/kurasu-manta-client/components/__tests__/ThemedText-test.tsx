import * as React from 'react'
import renderer, { act } from 'react-test-renderer'

// Mock the dependencies
jest.mock('@/hooks/useColorScheme')
jest.mock('@/constants/Colors')

import { ThemedText } from '../ThemedText'

describe('ThemedText', () => {
  it('renders correctly', () => {
    let tree: ReturnType<typeof renderer.create> | null = null

    act(() => {
      tree = renderer.create(<ThemedText>Snapshot test!</ThemedText>).toJSON()
    })

    expect(tree).toMatchSnapshot()
  })
})
