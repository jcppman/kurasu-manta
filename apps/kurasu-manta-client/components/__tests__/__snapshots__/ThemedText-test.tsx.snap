// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemedText renders correctly 1`] = `null`;
